app/controllers/admin/images_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Admin::ImagesController < Admin::BaseController
+  before_action :find_edition
```

Comment:
Is this controller always going to be coupled to images that belong to an edition? It seems likely that's the case for now as we use mounts for images on other objects (promotional features etc.)

I'm assuming that will be the case here as the designs are so customised for the editions flow (has it's own tab etc.)

Just double checking as `find_edition` couples this controller to editions.

If that is the case though maybe we should call this `Admin::EditionImagesController` so it's a bit more explicit that it's only going to be used for images as lots of other objects have them too